app/views/sign_up/completions/show.html.erb

```diff
@@ -24,6 +24,18 @@
         sp: content_tag(:strong, decorated_session.sp_name),
       ) %>
 </p>
+<% if @multiple_factors_enabled %>
```

Comment:
Should this be inverted? In other words, are we expecting to show a banner if the user _does not_ have multiple factors set up?

Reply:
At first, it felt kind of tricky to name since we are seeking the false value. I will think this through some more--maybe `only-one-factor-enabled` or something that shows what the variable actually does